catalyst/contrib/models/simple_conv.py

```diff
@@ -0,0 +1,41 @@
+import torch
+from torch import nn
+from torch.nn.functional import max_pool2d, relu
+
+from catalyst.contrib.nn import Normalize
+
+
+class SimpleConv(nn.Module):
+    """
+    Simple convolutional network.
+    """
+
+    def __init__(self, input_channels: int, features_dim: int):
+        """
+        Args:
+            input_channels: number of channels in the 1st layer
+            features_dim: size of the output tensor
+        """
+        super(SimpleConv, self).__init__()
+        self.conv1 = nn.Conv2d(input_channels, 32, 3, 1)
```

Comment:
could we use nn.Sequential ? :)

Reply:
Are you talking about rewriting this exact architecture using nn.Seq?

Reply:
yup, I think it would be easier to understand

Reply:
done